app/_src/policies/traffic-metrics.md

```diff
@@ -54,7 +54,8 @@ spec:
     - name: prometheus-1
       type: prometheus
       conf:
-        skipMTLS: false
+        tls:
+          mode: activeMTLSBackend
```

Comment:
with this change the <= 2.3 version is the same as >= 2.4 version... it does not make sense for me